core.windows/src/org/netbeans/core/windows/view/ui/toolbars/ConfigureToolbarPanel.java

```diff
@@ -107,6 +107,7 @@ public void run() {
         validate();
         repaint();
         setCursor( Cursor.getDefaultCursor() );
+		explorerManager = new ExplorerManager();
```

Comment:
I wonder why this was moved here.

Reply:
But why? The field was initialised directly. Does it matter to initialise it here?